scripts/generate-user-data-overview.ts

```diff
@@ -988,8 +988,8 @@ function sheetThemaContent(resultsByUser: ResultsByUser): SheetData {
       );
     },
     KREFIA: (serviceResults: ServiceResults) => {
-      const deeplinks = serviceResults.KREFIA?.content.deepLinks;
-      return !!deeplinks.length ? deeplinks.length : '';
+      const deeplinks = serviceResults.KREFIA?.content?.deepLinks;
+      return !!deeplinks?.length ? deeplinks.length : '';
```

Comment:
Nitpick: Hi, ik zie dat alle andere services het op deze manier teruggeven. Minder logica, minder nadenken, meer beter:)
```suggestion
      return serviceResults.KREFIA?.content?.deepLinks?.length || '';
```